net/brave_search_agent/samples/search.ts

```diff
@@ -0,0 +1,37 @@
+import "dotenv/config";
+
+import { GraphData } from "graphai";
+import { graphDataTestRunner } from "@receptron/test_utils";
+import braveSearchAgentInfo from "../src/brave_search_agent";
+
+const graph_data: GraphData = {
+  version: 0.5,
+  nodes: {
+    input: {
+      value: { content: "Graph AI" },
```

Comment:
If you search for GraphAI, no spaces are needed.

Reply:
https://github.com/receptron/graphai-agents/pull/18/commits/22574eafa5262fafc1ad73b1db8562666607d440